packages/issm/package.py

```diff
@@ -1,62 +1,135 @@
-# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
-# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
 #
 # Copyright 2023 Angus Gibson
 # Modified by Justin Kin Jun Hew, 2025
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
+import sys
 
 
 class Issm(AutotoolsPackage):
     """Ice-sheet and Sea-Level System Model"""
 
     homepage = "https://issm.jpl.nasa.gov/"
-    git = "https://github.com/ISSMteam/ISSM.git"
+    git = "https://github.com/ACCESS-NRI/ISSM.git"
 
-    version("develop", branch="main")
-    version("4.24", sha256="0487bd025f37be4a39dfd48b047de6a6423e310dfe5281dbd9a52aa35b26151a")
-    
     maintainers("justinh2002")
 
+    version("upstream", branch="main", git="https://github.com/ISSMteam/ISSM.git")
+    
+    version("main", branch="main", git="https://github.com/ACCESS-NRI/ISSM.git")
+    version("access-development", branch="access-development", git="https://github.com/ACCESS-NRI/ISSM.git")
```

Comment:
I forgot to mention that you should set `preferred=True` to one of the versions. Otherwise it's going to default to `upstream`.